scripts/album.js

```diff
@@ -352,6 +352,77 @@ album.setDescription = function(albumID) {
 
 };
 
+album.setLicense = function(albumID) {
+
+	let oldLicense = album.json.license;
+
+	const callback = function() {
+		$('select#license').val(album.json.license === '' ? 'none' : album.json.license);
+		return false;
+	};
+
+	const action = function(data) {
+
+		let license = data.license;
+
+		basicModal.close();
+
+		if (visible.album()) {
+			album.json.license = license;
```

Comment:
this update should be done in after:
```js
if (data!==true) lychee.error(null, params, data)
```
something in the line of:
```js
if (data!==true) {
	lychee.error(null, params, data)
}
else
{
		if (visible.album()) {
			album.json.license = license;
			view.album.license()
		}
}
```

In such way you update the json only if you are sure that it has been modified in the server side.